source/leuven/services/openondemand.rst

```diff
@@ -191,136 +241,308 @@ of using any of the other partitions. There are however some limitations on the
 
 This is put in place to ensure that these resources are kept for their original purpose, namely the interactive work.
 
-If for some reason some of these limitations are too strict for you, or you need resources that are not available on the interactive nodes (e.g. a full GPU, big 
-memory nodes), you can always request nodes from another partition. Remember however that these interactive apps are not meant for running full jobs. If you
-indeed need multiple nodes or full GPUs to test your code/program, go ahead and request the resources for your interactive app. In the case that you have passed
-the testing phase and you want to start conducting experiments, we recommend that you make the switch to batch jobs instead, as they will not require
-your presence to start your code.
+If for some reason some of these limitations are too strict for you, or you need resources that are not available on
+the interactive nodes (e.g. a full GPU, big memory nodes), you can always request nodes from another partition.
+Remember however that these interactive apps are not meant for running full jobs.
+If you indeed need multiple nodes or full GPUs to test your code/program, go ahead and request the resources for
+your interactive app from a more suitable partition.
+In the case that you have passed the testing phase, and you want to start conducting experiments,
+we recommend that you make the switch to batch jobs instead, as they will not require your presence to start your code.
 
 .. _interactive_shell:
 
 Interactive shell
```

Comment:
Perhaps we should shortly explain the issue with the module loading here as well (https://gitea.icts.kuleuven.be/coz/ood/issues/13)